openquake/commonlib/datastore.py

```diff
@@ -252,6 +252,19 @@ def export_path(self, relname, export_dir=None):
             export_dir = self.export_dir
         return os.path.join(export_dir, newname)
 
+    def build_fname(self, prefix, rlz, fmt, export_dir=None):
+        """
+        Build a file name from a realization, by using prefix and extension.
+
+        :param rlz: a realization object
+        :param prefix: the prefix to use
```

Comment:
Params rlz and prefix are swapped in the docstring, with respect to the signature